### DISCUSSION TITLE
Q&A

### DISCUSSION THREAD

**Discussion by stormbeforesunsetbee** [Musshal/ci4-sikromo#2] [General]
Sal, kie ws dadi rung? https://github.com/Musshal/ci4-sikromo/projects/1#card-58276064

**Comment by faishaldev** [comment]
durung, kwe mung contoh project cuy wkwk, ora pasti tak terapna, aku esih fokus nggarap ng fungsi liane

**Reply by stormbeforesunsetbee** [reply]
Oke. Sal, maaf jebule aku minggu kie akeh jadwal. Aku tak bantu gawekna logo bae ya

**Reply by faishaldev** [reply]
[
![20210123_134705](https://user-images.githubusercontent.com/72973311/113802887-aafbb800-9785-11eb-8dd3-3058c0db3518.jpg)
![IMG-20210117-WA0020](https://user-images.githubusercontent.com/72973311/113802896-b0f19900-9785-11eb-955d-485bcca6d0af.jpg)

](url)

santai bae pan, nek mbantu nggawe logo, kye aku ana contoh seko cv romo ne, logone kaya kie pan

**Comment by stormbeforesunsetbee** [comment]
Sal, fitur migrasine ws dadi urung?

**Reply by faishaldev** [reply]
durung pan, sesuk tak garap

**Reply by stormbeforesunsetbee** [reply]
Nice!
